src/troute-routing/troute/routing/compute.py

```diff
@@ -254,6 +298,7 @@ def compute_nhd_routing_v02(
     waterbody_type_specified,
     subnetwork_list,
     flowveldepth_interorder = {},
+    from_files = True,
```

Comment:
At line 306, there is an error saying "ValueError: could not convert string to float: '01010000,01010000,01010000,01010000' .  When I checked param_df's gages column, I found

(Pdb) test.gages
key
302    01010000,01010000,01010000,01010000
Name: gages, dtype: object

I think a single stream take a single gage at most, but it seems weird.